guides/source/7_2_release_notes.md

```diff
@@ -21,6 +26,252 @@ guide.
 Major Features
 --------------
 
+### Development Containers configuration for applications
+
+A [development container](https://containers.dev/) (or dev container for short) allows you to use a container
+as a full-featured development environment.
+
+Rails 7.2 adds the ability to generate a development container configuration for your application. This configuration
+includes a `.devcontainer` folder with a `Dockerfile`, a `docker-compose.yml` file, and, a `devcontainer.json` file.
+
+By default the dev container contains the following:
+
+* A redis container for use with Kredis, Action Cable etc.
+* A database (SQLite, Postgres, MySQL or MariaDB)
+* A Headless chrome container for system tests
+* Active Storage configured to use the local disk and with preview features working
+
+To generate a new application with a development container, you can run:
+
+```bash
+rails new myapp --devcontainer
+```
+
+For existing applications, a `devcontainer` command is now available:
+
+```bash
+rails devcontainer
+```
+
+For more information, see the [Getting Started with Dev Containers](https://edgeguides.rubyonrails.org/getting_started_with_devcontainer.html) guide.
+
+### Add browser version guard by default
+
+Rails now adds the ability to specify the browser versions that will be allowed to access all actions
+(or some, as limited by `only:` or `except:`).
+
+Only browsers matched in the hash or named set passed to `versions:` will be blocked if they're below the versions
+specified.
+
+This means that all other browsers, as well as agents that aren't reporting a user-agent header, will be allowed access.
```

Comment:
```suggestion
This means that all other unknown browsers, as well as agents that aren't reporting a user-agent header, will be allowed access.
```

I had to read this twice to understand that this sentence is actually correct in relation to the one above.